src/language/pythonLanguage.ts

```diff
@@ -1,30 +1,67 @@
 import { StringOrRegExp } from '@cucumber/cucumber-expressions'
+import { RegExps } from '@cucumber/cucumber-expressions/dist/cjs/src/ParameterType'
 
-import { Language, TreeSitterSyntaxNode } from './types.js'
+import { childrenToString, filter, NO_QUOTES } from './helpers.js'
+import { Language, NodePredicate, TreeSitterSyntaxNode } from './types.js'
 
 export const pythonLanguage: Language = {
-  toParameterTypeName() {
-    throw new Error('Unsupported operation')
+  toParameterTypeName(node: TreeSitterSyntaxNode) {
+    return node.text.slice(1, -1)
```

Comment:
Could you please check the node type here and throw an error if it's not the expected type? See https://github.com/cucumber/language-service/blob/5ebb00b14b69817d543a9eb6b58a6ad025894092/src/language/rustLanguage.ts#L4-L19